ionic2/app/models/pokemon.ts

```diff
@@ -4,8 +4,8 @@ import { ConfigService } from '../services/config.service';
 const configServiceInjector = ReflectiveInjector.resolveAndCreate([ConfigService]);
 const config: ConfigService = configServiceInjector.get(ConfigService);
 
-export type PokemonGender = 'm' | 'f' | 'g' | 'h';
-export type PokemonAttackCategory = 'fast' | 'special';
+export type PokeGender = 'm' | 'f' | 'g' | 'h';
+export type PokeAttackCategory = 'fast' | 'special';
```

Comment:
Oh, we actually use it in the apiservice ... of course the github search for `PokeAttackCategory` didnt return any results because that's only in the pull request ... ;)